editor/post-schedule/clock.js

```diff
@@ -12,6 +17,7 @@ function PostScheduleClock( { is12Hour, selected, onChange } ) {
 	const updateHours = ( event ) => {
 		const value = parseInt( event.target.value, 10 );
 		if (
+			! isInteger( value ) ||
```

Comment:
Simpler to do `isNaN( value )`? Here and below.

Reply:
I wanted to guard against `1.345`

Reply:
But `parseInt` wouldn't return a float?

```js
parseInt( '1.345', 10 )
// 1
```

Reply:
Right! I missed that :)